src/triggers/CBL_Attitud_UpdateDataCbl2.trigger

```diff
@@ -0,0 +1,49 @@
+trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
+    Set<Id> att_Ids = new Set<Id>();
+    Set<Id> cbl_Ids = new Set<Id>();
+    
+    //get a list with Baseline id related to attitude rows inserted
+    for (CBL_Attitude__c newRow : Trigger.new) {
+        att_Ids.add(newRow.Id);
+        cbl_Ids.add(newRow.Baseline__c);
+    }
+    
+    //get a list with attitude rows completed (field Baseline__r.Farmer__c)
+    for(List<CBL_Attitude__c> atts : [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c 
```

Comment:
Yes, it's because of conflicts. You need to first manually merge the latest
CI branch code into your branch, resolve conflicts, and then update the
pull request.

Luke.
ᐧ


Luke Kyohere

Group Manager, Software Development

Grameen Foundation <http://www.grameenfoundation.org/>

Mobile: 646-321-2691

lkyohere@grameenfoundation.org | Skype: luke1kyohere

Facebook <https://www.facebook.com/StopPovertyNow> | Twitter
<https://twitter.com/grameenfdn> | YouTube
<https://www.youtube.com/user/grameen?blend=6&ob=5>


<https://www.youtube.com/user/grameen?blend=6&ob=5>

*Connecting the World's Poor to Their Potential*


On Fri, Mar 6, 2015 at 4:30 PM, David Mugume <notifications@github.com>
wrote:

> In src/triggers/CBL_Attitud_UpdateDataCbl2.trigger
> <https://github.com/grameenfoundation/applab-ug.salesforce/pull/47#discussion_r25979789>
> :
>
> > @@ -0,0 +1,49 @@
> > +trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
> > +    Set<Id> att_Ids = new Set<Id>();
> > +    Set<Id> cbl_Ids = new Set<Id>();
> > +
> > +    //get a list with Baseline id related to attitude rows inserted
> > +    for (CBL_Attitude__c newRow : Trigger.new) {
> > +        att_Ids.add(newRow.Id);
> > +        cbl_Ids.add(newRow.Baseline__c);
> > +    }
> > +
> > +    //get a list with attitude rows completed (field Baseline__r.Farmer__c)
> > +    for(List<CBL_Attitude__c> atts : [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c
>
> Ah Ok. I get it. It seems to have been because the pullrequest had
> conflicts and merging was disabled
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grameenfoundation/applab-ug.salesforce/pull/47/files#r25979789>
> .
>